### PR TITLE
Sb 43 44

### PR DESCRIPTION
**Pull request by snreynolds**
## Related Issue

Which issue does this pull request resolve?

### CODE REVIEW

File: src/libraries/Pool.sol [hunk 1]
```diff
@@ -325,7 +325,7 @@ library Pool {
             if (params.sqrtPriceLimitX96 >= slot0Start.sqrtPriceX96()) {
                 PriceLimitAlreadyExceeded.selector.revertWith(slot0Start.sqrtPriceX96(), params.sqrtPriceLimitX96);
             }
-            if (params.sqrtPriceLimitX96 < TickMath.MIN_SQRT_PRICE) {
+            if (params.sqrtPriceLimitX96 <= TickMath.MIN_SQRT_PRICE) {
```
Comment (hensha256) [inline src/libraries/Pool.sol:328]: Note: This is an unrelated change to sb 43 and 44. The relevant changes are in TickMath.sol

**Comment by snreynolds** [comment]
https://github.com/Uniswap/v4-core/pull/867
